x-pack/plugins/security/server/authorization/actions/alerting.ts

```diff
@@ -27,6 +32,10 @@ export class AlertingActions {
       throw new Error('consumer is required and must be a string');
     }
 
-    return `${this.prefix}${alertTypeId}/${consumer}/${operation}`;
+    if (!alertingType || !isString(alertingType)) {
```

Comment:
nit.
Is it worth referring to these as `alertingEntity` rather than `alertingType` to avoid confusion with _alertType_?

Reply:
Renamed to entity in this commit: https://github.com/elastic/kibana/pull/99078/commits/bfbfae406722dc86bbfa235c16e618228c3f6ebe

Reply:
Thanks Ying :)